edgelet/edgelet-core/src/settings.rs

```diff
@@ -191,6 +194,9 @@ pub struct Settings<T> {
     #[serde(default, skip_serializing)]
     #[cfg_attr(not(debug_assertions), serde(skip_deserializing))]
     pub endpoints: Endpoints,
+    pub edge_ca_cert: Option<String>,
+    pub edge_ca_key: Option<String>,
+    pub trust_bundle_cert: Option<String>,
```

Comment:
Move these above the `endpoints` field to be consistent with the other services' configs.

Reply:
Done